samples/create_deployment.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020, Google LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Create a Game Servers Deployment.
+ * @param {string} projectId string project identifier
+ * @param {string} deploymentId unique identifier for the new Game Server Deployment
+ */
+function main(projectId = 'YOUR_PROJECT_ID', deploymentId = 'DEPLOYMENT_ID') {
```

Comment:
Sorry I missed this in the original review, this needs to be `async function main(...`

Reply:
Done.